package.json

```
@@ -1,7 +1,7 @@
 {
     "name": "nrfconnect",
-    "version": "3.7.1",
-    "description": "nRF Connect for PC",
+    "version": "3.8.0",
```

I suggest that we call the version `3.8.0-pre1` before release. 

```suggestion
    "version": "3.8.0-pre1",
```